### PR TITLE
add brief core structure + item comment implementation

### PR DESCRIPTION
**Pull request by samjo-nyang**
- add new permission classes for items and comments
- add a router to the core app
- add two new log groups - item and comment
- remove unused views.py code
- add viewsets for items and comments
- implements creating, listing, and deleting comments

### CODE REVIEW

File: sparcs09-api/apps/core/viewsets/comment.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        }, status=405)
+
+    # [POST] /comments/
+    def create(self, request):
+        return Response({
+            'detail': 'Creating is not allowed here.',
+        }, status=405)
+
+    # [GET] /comments/<pk>/
+    def retrieve(self, request, pk=None):
```
Comment (ktk1012) [inline sparcs09-api/apps/core/viewsets/comment.py]: 크게 쓸 일이 없는 API 일거 같으니 막아주세용

Comment (ktk1012) [inline sparcs09-api/apps/core/viewsets/comment.py]: 쓸 일이 없는 API이니 막아주세용

Comment (samjo-nyang) [inline sparcs09-api/apps/core/viewsets/comment.py]: 확인해주세요!

Comment (ktk1012) [inline sparcs09-api/apps/core/viewsets/comment.py]: 확인했습니다.

**Pull request merged**